src/components/cards/CardItem.vue

```diff
@@ -148,6 +151,10 @@ export default {
 		opacity: 0;
 	}
 
+	.current-card {
+		box-shadow: 0 0 3px 0 grey !important;
```

Comment:
I'd say we use the same color but a bit bigger shadow:

```suggestion
		box-shadow: 0 0 5px 0 var(--color-box-shadow);
```

Reply:
I need the important Expression otherwise it won't work
Could we go up to 6px for better recognition?

Reply:
6px is also fine for me
try to move it into the actual .card section and use &.current-card to apply the class selector on top of that, so it gets applied after the regular card style in line 159